osquery/CMakeLists.txt

```diff
@@ -280,7 +280,11 @@ if(NOT ${OSQUERY_BUILD_SDK_ONLY})
     DESTINATION bin COMPONENT main)
 
   # make install (config files)
-  install(FILES "${CMAKE_SOURCE_DIR}/tools/deployment/osquery.example.conf"
+  if(APPLE)
+    install(FILES "${CMAKE_SOURCE_DIR}/tools/deployment/osquery.example.osx.conf"
```

Comment:
I'd rather not maintain this second file. Then we'll need to remember to update both if changes are ever needed. The only difference is the install `${PREFIX}` or "home". Can we just call that out in the `osquery.example.conf` somewhere nearby the packs in the comments?